src/lib/index.ts

```diff
@@ -203,11 +183,7 @@ export function displayTimeLeft(created_at: number, isMobile = false) {
 	const daysLeft = Math.floor(timeLeft / 60 / 60 / 24);
 	const hoursLeft = Math.floor((timeLeft - daysLeft * 60 * 60 * 24) / 60 / 60);
 
-	if (isMobile && daysLeft > 0) {
-		return `${daysLeft} days left`;
-	} else if (isMobile && hoursLeft > 0) {
-		return `${hoursLeft} hours left`;
-	} else if (daysLeft > 0 && hoursLeft > 0) {
```

Comment:
Why do you want to change this? It doesn't seem related to this PR.

Reply:
Oh I missed it, I would have put it in the fix PR.